internal/pkg/describe/lb_web_service.go

```diff
@@ -219,39 +226,131 @@ func (d *WebServiceDescriber) URI(envName string) (string, error) {
 	return uri.String(), nil
 }
 
-// EnvVars contains serialized environment variables for a service.
-type EnvVars struct {
+// EnvVar contains serialized environment variables for a service.
+type EnvVar struct {
 	Environment string `json:"environment"`
+	Container   string `json:"container"`
 	Name        string `json:"name"`
 	Value       string `json:"value"`
 }
 
-type envVars []*EnvVars
+type envVars []*EnvVar
 
 func (e envVars) humanString(w io.Writer) {
-	fmt.Fprintf(w, "  %s\t%s\t%s\n", "Name", "Environment", "Value")
-	var prevName string
-	var prevValue string
+	headers := []string{"Name", "Container", "Environment", "Value"}
+	fmt.Fprintf(w, "  %s\n", strings.Join(headers, "\t"))
+	fmt.Fprintf(w, "  %s\n", strings.Join(underline(headers), "\t"))
+	sort.SliceStable(e, func(i, j int) bool { return e[i].Environment < e[j].Environment })
+	sort.SliceStable(e, func(i, j int) bool { return e[i].Container < e[j].Container })
+	sort.SliceStable(e, func(i, j int) bool { return e[i].Name < e[j].Name })
+	// Pre-populate the previous map with dashes; otherwise, if fields are somehow empty, they get ditto symbols.
+	previous := map[string]string{
+		"name":        "-",
+		"container":   "-",
+		"environment": "-",
+		"valueFrom":   "-",
+	}
 	for _, variable := range e {
```

Comment:
What do you think of this:

```go
if len(e) > 0 {
  fmt.Printf(w, "  %s\n", strings.Join([]string{env.Name, env.Container, env.Environment, env.Value}, "\t"))
}
for prev, cur := 0, 1; cur < len(e); prev += 1; cur +=1; {
  cols := []string{env.Name, env.Container, env.Environment, env.Value}
  if e[prev].Name == e[cur].Name {
      cols[0] = dittoSymbol
  }
  // ... same for the remaining fields.
  fmt.Printf(w, "  %s\n", strings.Join(cols, "\t"))
}
```

Reply:
LOVE it. That is such a better way to refer to the previous values! At one point I had tried to make a map of slices, when really THIS is what I was trying to do. Thank you!